environments/illumina/environment.yml

```diff
@@ -1,15 +1,14 @@
-name: artic-ncov2019-illumina
+name: gms-artic-illumina
```

Comment:
If we change this we need to change these lines in the Docker file?
```
COPY --from=condabuild /opt/conda/envs/artic-ncov2019-illumina /opt/conda/envs/artic-ncov2019-illumina
ENV PATH=/opt/conda/envs/artic-ncov2019-illumina/bin:$PATH
```

i.e. "artic-ncov2019-illumina" should be replaced with "gms-artic-illumina"?

Reply:
This could be the reason. I revert back to "artic-ncov2019-illumina" for` environment.yml` and rebuilt the container, pushed to gms dockerhub with same name.

Reply:
ok, I pull that container and try again! but still I think you should expand the illumina section in `nextflow.config` with:
```
withName:typeVariants {
            container = "docker://genomicmedicinesweden/gms-artic-illumina:latest"
}
```

Reply:
added to nextflow.config